README.md

```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
 
 And you're done! This mini-tutorial doesn't support the leaderboard, though. You also may wish to set up a simple
 docker compose file. [Join the discord](https://valk.sh/discord) to learn more!
+
+
+# Using Docker Compose
+You can grab the file from [here](<https://github.com/PBOwner/xpd/blob/prod/docker-compose.yml>)
+
+## Env File:
+You can grav the variables from [here](<https://github.com/PBOwner/xpd/blob/prod/.env.example>)
+
+Make sure you replace `<token>` and both `<db_pass>` with your own bot token and database password for postgres. You do **NOT** need to configure postgres, just set those variables up and it will configure for you.
+
+## Finally, start the docker with:
+```bash
+docker compose up
+```
+If there are any errors, ping rosilla07 or valkyrie_pilot on the official discord [here](<https://discord.gg/sGX2EhceNc>)
```

Comment:
I have a link for this already: [https://valk.sh/discord](https://valk.sh/discord). Also, I'm not sure how I feel about listing you as a potential help source. I'm not necessarily adverse to it, but it does seem like something worth discussing